spec/integration/geo_aardvark_config_spec.rb

```diff
@@ -296,18 +296,6 @@
     end
   end
 
-  context 'with a scanned map that has a georeferenced map' do
-    let(:druid) { 'gm036df2527' }
-
-    it 'maps the georeferenced status' do
-      expect(result['gbl_georeferenced_b']).to eq true
-    end
-
-    it 'populates the link to the georeferenced map in dct_relation_sm' do
```

Comment:
Is dct_relations_sm modified in this PR? Might be missing something.

Reply:
No, but this change was apparently part of the commit that got reverted, so it also was reverted...I'll see if I can fix that with another commit

Reply:
Well then it's my bad for a messy commit! We can add it later if easier with the git.

Reply:
I was able to fix it and restore the fixture for this test also. Makes the PR a little cleaner to review.